spec/lib/miq_automation_engine/models/miq_ae_yaml_import_export_spec.rb

```diff
@@ -197,8 +195,8 @@ def assert_import_failure_with_missing_file(import_options)
       end
 
       def assert_all_domains_imported(export_options, import_options)
-        export_model(ALL_DOMAINS, export_options)
-        reset_and_import(@export_dir, ALL_DOMAINS, import_options)
+        export_model(MiqAeYamlImportExportMixin::ALL_DOMAINS, export_options)
```

Comment:
lol, rspec is so much magic, I'm fine with `MiqAeYamlImportExportMixin::ALL_DOMAINS` all over here, let rails autoload it.